Recolor.php

```diff
@@ -53,8 +53,12 @@ public function image($imageName, $newFilename)
             }
         }
 
-        imagepng($image, $this->imageFolder . $newFilename);
-        imagedestroy($image);
+        if (imagepng($image, $this->imageFolder . $newFilename)) {
+            imagedestroy($image);
+            return "New image was created and saved.";
```

Comment:
Now this is me taking some liberty for the sake of demonstration. Often you might want to use the value of the new file right? Perhaps you want to attach it to an email? Log it somewhere? Just tell the user it worked?

So we should probably `return` something from the class method to whatever has called it. In this instance the `imagepng()` method returns a boolean of `true` or `false`, so by wrapping that in an `if()` we're telling PHP to look for a `truthy` value, in order to execute the `if` block, which is to return a string saying stuff worked.